composer.json

```diff
@@ -26,7 +26,6 @@
         "doctrine/migrations": "^3.7",
         "doctrine/orm": "^2.14.0",
         "firebase/php-jwt": "^6.3",
-        "google/recaptcha": "^1.2",
```

Comment:
probably worth removing from `composer.lock` as well. Could just run `composer remove google/recaptcha` or we could just do `composer update` since we don't have a schedule to do updates and it's worth doing them all from time to time anyway.

Reply:
Yep, thought I did the former before – think it got lost in a later merge resolve